etc/kernel-launchers/kubernetes/scripts/launch_kubernetes.py

```diff
@@ -1,3 +1,4 @@
+#!/opt/conda/bin/python
```

Comment:
This is interesting.  What does this do?  Does `launch_custom_resource.py` need this as well?

When called via `python launch_kubernetes.py` does it have an influence on _which_ python is used?  (Just curious)

Reply:
Hmm - this does seem like a _fallback_ mechanism for when the .py script is not invoked using `python`: https://stackoverflow.com/questions/6908143/should-i-put-shebang-in-python-scripts-and-what-form-should-it-take#:~:text=14%20Answers&text=The%20shebang%20line%20in%20any,manager%20(when%20configured%20properly).

Was this needed for testing the script or something since every invocation from a `kernel.json` uses `python path-to-launch_kubernetes.py`?

At any rate, I guess it doesn't hurt, but we should be consistent (relative to the `launch_custom_resource.py`).

Reply:
Good catch @kevin-bates we need this on the `launch_custom_resource.py` as well.

Reply:
yes, need to add in launch_custom_resource.py. 
Also, without the change, I got the error when invoking it via Enterprise Gateway.